packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec

```diff
@@ -109,6 +81,9 @@ mv %{buildroot}%{gem_instdir}/settings.d/discovery.yml.example \
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Thu Mar 07 2024 nofaralfasi <nalfassi@redhat.com> 1.0.5-1
+- Update to 1.0.5-1
```

Comment:
@evgeni  Any idea why it didn't jump to `1.0.5-10`?

Reply:
And what about line 5? `%global foreman_proxy_min_version 1.24` is correct? (used to be 1.25)

Reply:
That comes from https://github.com/theforeman/foreman-packaging/blob/7511869cc09cf038e6aaa503a08b83412c32f46a/gem2rpm/smart_proxy_plugin.spec.erb#L15, no idea which version your plugin needs :)